admin/app/controllers/solidus_admin/backend_auth_support.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module SolidusAdmin::BackendAuthSupport
```

Comment:
Shouldn't we also define the `spree_current_user` method? Or is it already available from somewhere else?

Reply:
It's coming from `spree_auth_devise`

Reply:
As above, do you think a comment will make things clearer?